### PR TITLE
feat(presentational): Add box component

### PR DESCRIPTION
**Pull request by agory**
It's simple component that use the box class of Bulma. We can use it to separate an article/section from the rest of the page. It adds a light box-shadow and rounded borders.
https://bulma.io/documentation/elements/box/

### CODE REVIEW

File: src/web/components/Box/box.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Container from './';
```
Comment (mfrachet) [inline src/web/components/Box/box.test.js]: Box or Container ?

**Comment by mfrachet** [comment]
Same remark as on the other one, you can now include storybooks inside your PR with presentationals :)

**Comment by mfrachet** [comment]
For my information, when / where would you like to use such a component ?

File: src/web/components/Box/box.test.js (hunk 1 above)
Comment (agory) [reply src/web/components/Box/box.test.js]: It's a box. sorry for the copy/paste mistake

**Comment by agory** [comment]
It's simple component that use the box class of Bulma. We can use it to separate an article/section from the rest of the page. It adds a light box-shadow and rounded borders.

File: src/web/components/Container/Container.test.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import Container from './';
+import Box from './';
```
Comment (mfrachet) [inline src/web/components/Container/Container.test.js]: I think you're in the wrong file 😛

Comment (agory) [reply src/web/components/Container/Container.test.js]: Oh :hankey: . i'm a noob

**Pull request merged**